src/olympia/users/models.py

```diff
@@ -87,25 +87,30 @@ def widget_attrs(self, widget):
 
 class UserManager(BaseUserManager, ManagerBase):
 
-    def create_user(self, username, email, fxa_id=None):
+    def create_user(self, username, email, fxa_id=None, **kwargs):
         # We'll send username=None when registering through FxA to generate
         # an anonymous username.
         now = timezone.now()
         user = self.model(
-            username=username, email=email, fxa_id=fxa_id,
-            last_login=now)
+            username=username,
+            email=email,
+            fxa_id=fxa_id,
+            last_login=now,
+            **kwargs
```

Comment:
Did you add this to support setting `display_name`?

Reply:
Yes